armada-ths/ais
# Optional student_profile fields

Added facebook_profile, linkedin_profile and phone_number fields to student_profile.

Also implemented them as optional fields in student_profile api.

To test:
* make sure tests run
* create a manual PUT request to localhost:8000/api/student_profile?student_id={A_NUMBER}
with body = json'{'nickname':'NAME', ...}, where ... can be one or more of the optional fields (all of them are strings).
* make sure you get 'Profile updated successfully' response
* goto localhost:8000/api/student_profile?student_id={THE_SAME_NUMBER}
* make sure you see the information from the previous PUT request
